src/main/java/com/ays/auth/model/enums/AysUserType.java

```diff
@@ -5,6 +5,7 @@
  */
 public enum AysUserType {
 
+    SUPER_ADMIN,
     ADMIN,
     USER
 
```

Comment:
It would be very useful for us to add a default user with the role of `SUPER_ADMIN` in the database, similar to `ADMIN` and `USER`, for the purpose of testing and conducting checks. However, since this user won't have an organization, we need to remove the `NOT NULL` constraint on the `ORGANIZATION_ID` column in the `AYS_ADMIN_USER` table. Admin users won't have an organization.

Reply:
Ok. I revised the code. Can you look through it again?

Reply:
> No action has been taken regarding this comment. Can we review it again?

@Rapter1990 The validity of my previous comment regarding the first item still stands. It appears that no developments have been made. Can you please verify it?